packages/utils/src/utils.ts

```diff
@@ -107,9 +116,18 @@ export function setStyles(target: VNode | InlineFormat, styles: Record<string, s
         stylesArray.push([key, styles[key]].join(': '));
     }
     if (stylesArray.length) {
-        target.attributes.style = stylesArray.join('; ') + ';';
-    } else {
-        delete target.attributes.style;
+        if (target.modifiers.get(Attributes)) {
+            target.modifiers.get(Attributes).set('style', stylesArray.join('; ') + ';');
+        } else {
+            const attributes = new Attributes();
+            attributes.set('style', stylesArray.join('; ') + ';');
+            target.modifiers.append(attributes);
+        }
+    } else if (target.modifiers.get(Attributes)) {
+        target.modifiers.get(Attributes).remove('style');
+        if (!Object.keys(target.modifiers.get(Attributes).value).length) {
+            target.modifiers.remove(Attributes);
+        }
```

Comment:
These utils will later be replaced by the `CSSStyle` modifier. The parser will do this conversion once then we'll be able to use the modifier directly and never manipulate "style" strings anymore until the renderer.

Reply:
... and that will be goooood.